src/ccnl-core/src/ccnl-relay.c

```diff
@@ -999,7 +1005,7 @@ ccnl_cs_add(struct ccnl_relay_s *ccnl, struct ccnl_content_s *c)
 {
     struct ccnl_content_s *content;
 
-    content = ccnl_content_add2cache(ccnl, c);
+    content = ccnl_content_add2cache(ccnl, c, NULL);
```

Comment:
Currently just calling this with tclass=NULL to make it build.

Eventually, I would want the ability to actually create content with specific service classes from scratch, which would require an extension here and in the CCNL main loop.